src/databricks/labs/dqx/check_funcs.py

```diff
@@ -12,6 +13,16 @@
 from databricks.labs.dqx.rule import register_rule
 from databricks.labs.dqx.utils import get_column_as_string, is_sql_query_safe, normalize_col_str
 
+_IPV4_OCTET = r"(25[0-5]|2[0-4]\d|1\d{2}|[1-9]?\d)"
+CIDR_PREFIX_LENGTH_GROUP = 5
```

Comment:
this is only used in the `_convert_cidr_to_bits_and_prefix`. I would kept it local, or do we plan to use it somewhere else?